appcelerator/titanium_mobile
# [TIMOB-25376] Android: Fixed bug where scrolled in ListView rows forget their width/height

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25376

**Test:**

1. Run test code attached to [TIMOB-25376](https://jira.appcelerator.org/browse/TIMOB-25376) on Android.
2. Scroll down to the bottom of the ListView.
3. Verify that none of the rows have grown to the window's height. (See case's screenshots.)
